com.ozmartians.VidCutter.json

```diff
@@ -35,6 +42,30 @@
             "type": "archive",
             "url": "https://pypi.python.org/packages/9c/1d/4544708aaa89f26c97cc09450bb333a23724a320923e74d73e028b3560f9/PyOpenGL-3.1.0.tar.gz",
             "sha256": "9b47c5c3a094fa518ca88aeed35ae75834d53e4285512c61879f67a48c94ddaf"
+        }],
+        "modules": [{
+            "name": "freeglut",
+            "buildsystem": "simple",
```

Comment:
Why not `"buildsystem": "cmake-ninja"` ?

Reply:
because i'm not too familiar with it but will give it a try for cmake in future (the freeglut + glu modules have been removed for the manifest and not needed now)

thanks!